### DISCUSSION TITLE
buddhism/2021/favorites/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#133] [General]
# buddhism/2021/favorites/

https://shanenull.com/buddhism/2021/favorites/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/heart_sutra/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/seven/

**Comment by shane0** [comment]
Snippets from Urgyen's book As It Is Vol 2

"You could also practice the preliminaries with simply a good and sincere attitude, and this alone will definitely purify your negative karma. "

"I hear that nowadays several masters first give the pointing-out instruction, introducing people to the main point of the practice, and afterwards teach the preliminaries."

"When masters and disciples do not have a lot of time to spend together, there is no opportunity to go through the whole sequence of instructions. I usually also give the whole set of teachings in completeness, all at once."

"...we are now in the Dark Age and there is the prediction that, "At the end of the Dark Age, the teachings of Secret Mantra will blaze forth like wildfire." Secret Mantra here refers to Mahamudra and Dzogchen."
